cards.tex

```diff
@@ -2,64 +2,6 @@
 
 \begin{document}
 
-\Category{References}
-
-\input{references.tex}
-
-\Category{Rules}
-
-\input{cards/rules/bulk.tex}
-
-\Category{Actions}
-
-\input{cards/actions/demoralize.tex}
-
-\Category{Conditions}
-
-\input{cards/conditions/encumbered.tex}
-\input{cards/conditions/frightened.tex}
-
-\Category{Items}
-
-\input{cards/items/backpack.tex}
-\input{cards/items/bedroll.tex}
-\input{cards/items/breastplate.tex}
-\input{cards/items/chalk.tex}
-\input{cards/items/crossbow.tex}
-\input{cards/items/dagger.tex}
-\input{cards/items/flint-and-steel.tex}
-\input{cards/items/hat-of-disguise.tex}
-\input{cards/items/healers-tools.tex}
-\input{cards/items/healing-potion.tex}
-\input{cards/items/longsword.tex}
-\input{cards/items/rations.tex}
-\input{cards/items/repair-kit.tex}
-\input{cards/items/rope.tex}
-\input{cards/items/scimitar.tex}
-\input{cards/items/shortbow.tex}
-\input{cards/items/smoking-sword.tex}
-\input{cards/items/soap.tex}
-\input{cards/items/steel-shield.tex}
-\input{cards/items/thieves-tools.tex}
-\input{cards/items/torch.tex}
-\input{cards/items/wand-of-heal.tex}
-\input{cards/items/waterskin.tex}
-
-\Category{Spells}
-
-\input{cards/spells/acid-splash.tex}
-\input{cards/spells/burning-hands.tex}
-\input{cards/spells/chill-touch.tex}
-\input{cards/spells/detect-magic.tex}
-\input{cards/spells/feather-fall.tex}
-\input{cards/spells/force-bolt.tex}
-\input{cards/spells/heal.tex}
-\input{cards/spells/illusory-disguise.tex}
-\input{cards/spells/light.tex}
-\input{cards/spells/mage-hand.tex}
-\input{cards/spells/magic-missile.tex}
-\input{cards/spells/pest-form.tex}
-\input{cards/spells/ray-of-frost.tex}
-\input{cards/spells/shocking-grasp.tex}
```

Comment:
Moved to `cards/cards.tex` because the list of names could already be seen as Open Game Content and I want to put all of that in the `cards` folder.